pom.xml

```diff
@@ -30,7 +30,7 @@
   <groupId>org.apache.zookeeper</groupId>
   <artifactId>zookeeper</artifactId>
   <packaging>pom</packaging>
-  <version>2.6.0-SNAPSHOT</version>
+  <version>3.5.5-beta-SNAPSHOT</version>
```

Comment:
Should we move this to a separate PR? Seems it’s not highly related to the Markdown change here.

Reply:
What is 2.6.0 ?
It doesn't make sense

IMHO it is better to have the same version as the current version